config/rbac/cluster_bindings/manifest_clusterrole_binding.yaml

```diff
@@ -1,11 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: manager-rolebinding-manifest
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: manager-role-manifest
-subjects:
-  - kind: ServiceAccount
-    name: controller-manager
```

Comment:
not sure why this gets deleted entirely. Need to follow up on this.

Reply:
Seems to have been moved to the base permissions. Not sure why.

Reply:
These permissions used to be defined in a separate file. Now they have been moved to kubebuilder markers and are therefore included in the base permissions. So should be fine.